spec/requests/api/schemas/json/admin_user_list_response.json

```diff
@@ -1,118 +1,130 @@
 {
```

Comment:
Given that this is a documented API, I don't think we should introduce a breaking change as part of this PR that is prefixed `DEV`.

Reply:
yes I hesitated to do all of this... I would be very surprise anyone does something crazy with this admin api. But I can remove all this code and just skip the test.